blogs/network_events/index.md

```diff
@@ -0,0 +1,208 @@
+Detect unauthorized outbound connections: You can flag outbound traffic that violates egress rules.
+
+In the following example, there are three events, one for network policy, another for admin network policy and the third for egress firewall
+
+```yaml
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: default
+  namespace: iperf
+spec:
+  egress:
+  - to:
+      cidrSelector: 8.8.8.8/24
+    type: Allow
+  - to:
+      cidrSelector: 0.0.0.0/0
+    type: Deny
+```
+
+![egress firewall network events](./images/egress_fw_network_events.png)
+_Screen capture of the traffic flows in the NetObserv console plugin,
+with a network events column showing traffic allowed or denied by egress firewall rules_
+
+## Integration with the Network Observability stack
+
+- `eBPF Agent`:
+
+The agent manages a new `kprobe` hook through a configuration option.
+It also registers with the OVN observability library, allowing eBPF-generated events to be enriched
+into meaningful strings.
+
+- `Agent User Space`:
+
+The Agent user space handles the aggregation of events for processing.
+
+- `Flowlogs Pipeline`:
+
+Generated flows are transmitted over gRPC to the flowlogs-pipeline,
+where the flow content is enriched with Kubernetes metadata.
+
+- `Loki storage and Console-plugin`:
+
+The enriched flows are then sent to Loki storage, making them available for
+consumption by the NetObserv console plugin. The new events are displayed in the console.
+
+- `New Filtering Capability`:
+
+A filter is introduced to filter flows based on network event enrichment.
+
+![network events building blocks](./images/network_events_design.drawio.png)
+
+## Configuration
+
+- Enable the `OVNObservability` feature in the `FeatureGate` custom resource.
+
+```shell
+$ oc edit featuregates.config.openshift.io cluster
+```
+
+```yaml
+enabled:
+- name: OVNObservability
+```
+
+Note:
+ The above FeatureGate might be on by default at the time this feature becomes GA.
+
+- Enable Network events in the Network Observability `FlowCollector` custom resource.
+
+```yaml
+sampling: 1
+privileged: true
+features:
+- "NetworkEvents"
+```
+
+Note:
+
+- in the above example we set `sampling` to 1 to be able to capture all network events without missing any.
+
+- `privileged` is set to `true` because the `OVN observability` library needs to access local OVS socket and OVN databases.
+
+## Availability
+
+The networking events monitoring feature is available as Developer Preview for OCP 4.17 z-stream and NetObserv release 1.7,
+and is planned for GA in OCP 4.18 and NetObserv release 1.8.
+
+## Conclusion
+
+Relying on eBPF technology for network observability, particularly in environments governed by OVN network policies and egress firewalls,
+offers deep, real-time insights into network traffic.
+Users can monitor policy compliance, detect violations, and track detailed network flows.
+This helps you maintain a secure, high-performing network environment.
```

Comment:
What about suggesting an action to the reader as a final step ?

I usually put the following bloc:
```suggestion
## Conclusion

Relying on eBPF technology for network observability, particularly in environments governed by OVN network policies and egress firewalls,
offers deep, real-time insights into network traffic.
Users can monitor policy compliance, detect violations, and track detailed network flows.
This helps you maintain a secure, high-performing network environment.

## Feedback

We hope you liked this article !

Netobserv is an OpenSource project [available on github](https://github.com/netobserv).
Feel free to share your [ideas](https://github.com/netobserv/network-observability-operator/discussions/categories/ideas), [use cases](https://github.com/netobserv/network-observability-operator/discussions/categories/show-and-tell) or [ask the community for help](https://github.com/netobserv/network-observability-operator/discussions/categories/q-a).
```